src/utils/keyHandler.jsx

```diff
@@ -30,24 +36,34 @@ class KeyHandler extends Component {
 
   /* ------ Event Handlers ------ */
 
-  prvEventTargetIsRefDescendant(evt) {
+  prvEventTargetIsRefDescendant = (evt) => {
     const { getInputRef } = this.props;
     if (getInputRef) {
-      const refNode = ReactDOM.findDOMNode(getInputRef()); // eslint-disable-line max-len, react/no-find-dom-node
-      let node = evt.target;
-      let isDescendant = false;
-      do {
-        if (node === refNode) {
-          isDescendant = true;
+      let inputRef = getInputRef();
+      if (!_isArray(inputRef)) {
+        inputRef = [inputRef];
+      }
```

Comment:
I want keys bubbling up to either table (`cellGrid` or `cellGridFreeze`) to be handled seamlessly bu this component, so I added tuspport for `getInputRef` to return an array